assets/cross-sell-carousel.css

```diff
@@ -0,0 +1,88 @@
+@media (min-width: 751), (max-width: 2000px) {
+
+  .cross-sell-product-title {
+    /* show only 2 lines of text */
+    display: block;
+    text-overflow: ellipsis;
+    word-wrap: break-word;
+    overflow: hidden;
+    max-height: 3.6em;
+    line-height: 1.8em;
+    /* end */
+    padding: 0 10px;
+    font-size: 12px;
+  }
+
+  .glide__arrow--left {
+    left: -1em;
+  }
+
+  .glide__arrow--right {
+    right: -1em;
+  }
+}
+
+@media (max-width: 750px) {
+
+  .cross-sell-product-title {
+    height: 60px;
+  }
+
+  .glide__arrow--left {
+    left: 1em;
+  }
+
+  .glide__arrow--right {
+    right: 1em;
+  }
+}
+
+@media (max-width: 450px) {
```

Comment:
Some people are fine with being desktop first, but there is a big push to be mobile first, especially in big companies.

Instead of doing
 
doing max width, have your top css be mobile, and bottom version be desktop, using min-width, so that you are mobile first.